### DISCUSSION TITLE
× RangeError: Invalid time value react-redux app

### DISCUSSION THREAD

**Discussion by ValentinaPanic** [js-fns/date-fns#2295] [Q&A]
Hi, I keep getting this error and not sure how to fix it. I am working on my final project for Flatiron school and help is much needed.

`RangeError: Invalid time value
format
node_modules/date-fns/esm/format/index.js:373
  370 | var originalDate = toDate(dirtyDate);
  371 | 
  372 | if (!isValid(originalDate)) {
> 373 |   throw new RangeError('Invalid time value');
      | ^  374 | } // Convert the date in system timezone to the same date in UTC+00:00 timezone.
  375 | // This ensures that when UTC functions will be implemented, locales will be compatible with them.
  376 | // See an issue about UTC functions: https://github.com/date-fns/date-fns/issues/376
View compiled
DayCard
src/components/DayCard.js:11
   8 |   const newToday = new Date(today)
   9 | 
  10 |   const dayFormat = "yyyy-MM-dd"
> 11 |   const formattedDate = format( newToday,dayFormat)
     | ^  12 | console.log(formattedDate)
  13 |     return(
  14 |        <div >`

This is the component where the error triggers 

``` import React from "react";
 import { format } from "date-fns"
//  import format from 'date-fns/format'
import FoodCard from "./FoodCard";

const DayCard = ({days, today}) => {
     console.log(typeof today)
   const newToday = new Date(today)
 
   const dayFormat = "yyyy-MM-dd"
   const formattedDate = format( newToday,dayFormat)
 console.log(formattedDate)
     return(
        <div >
            <h3 style={{textAlign:"center"}}> Meals for {formattedDate}</h3>
           { days && days.filter(day => 
            
           day.attributes.date === formattedDate).map( day => { 
               return (day.attributes.foods.map(food =>{
                  return <FoodCard food={food} today={today}/> }
             ))
           })
                  
          }
        </div>
  
     )}


export default DayCard`
`today` is a prop sent from the parent component and it is grabbed from the redux store. When I console.log type of it says it is an object. but when I console.log formatted days I get what I want.

this is my GitHub repo [https://github.com/ValentinaPanic/baby-eats-frontend](url) if that will help to see the error. Thank you.

**Comment by yashmittalJTG** [comment]
facing same issue please let me know if you found the solution,

thanks

**Reply by ValentinaPanic** [reply]
I haven't find the way around it, I just went a different direction with my app. I am sorry

**Comment by yashmittalJTG** [comment]
You can use date-parse-format package
Thanks
